web_tree_color/README.md

```diff
@@ -0,0 +1,2 @@
+Web Tree Color
```

Comment:
You need to add a README.rst following https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst